functions/go/create-setters/createsetters/create_setters.go

```diff
@@ -88,6 +91,7 @@ func (cs *CreateSetters) Filter(nodes []*yaml.RNode) ([]*yaml.RNode, error) {
 			return nodes, err
 		}
 		cs.filePath = filePath
+		cs.preProcessScalarSetters()
```

Comment:
Why are you doing this for each node in for loop ? You should do it only once before the for loop.